flexmeasures_openweathermap/sensor_specs.py

```diff
@@ -15,23 +15,33 @@
 }
 
 
-owm_to_sensor_map = dict(
-    temp={
-        "name": "temperature",
-        "unit": "°C",
-        "event_resolution": timedelta(minutes=60),
-        "attributes": weather_attributes,
-    },
-    wind_speed={
-        "name": "wind speed",
-        "unit": "m/s",
-        "event_resolution": timedelta(minutes=60),
-        "attributes": weather_attributes,
-    },
-    clouds={
-        "name": "irradiance",
-        "unit": "kW/m²",
-        "event_resolution": timedelta(minutes=60),
-        "attributes": weather_attributes,
-    },
-)
+mapping = [
+    dict(
+        fm_sensor_name="temperature",
+        owm_sensor_name="temp",
+        unit="°C",
+        event_resolution=timedelta(minutes=60),
+        attributes=weather_attributes,
+    ),
+    dict(
+        fm_sensor_name="wind speed",
+        owm_sensor_name="wind_speed",
+        unit="m/s",
+        event_resolution=timedelta(minutes=60),
+        attributes=weather_attributes,
+    ),
+    dict(
+        fm_sensor_name="cloud cover",
+        owm_sensor_name="clouds",
+        unit="%",
+        event_resolution=timedelta(minutes=60),
+        attributes=weather_attributes,
+    ),
+    dict(
+        fm_sensor_name="irradiance",
```

Comment:
Maybe a comment here helps the reader. In save_forecasts_to_db, we catch this name and do the actual computation to get to the irradiance.

It doesn't happen when we store JSON, btw.

I'll create a ticket here to make such computations more generic. A function store?

Reply:
Thanks. I believe that would be quite useful.